src/configure-feature.js

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
+import { curry } from 'ramda';
```

Comment:
Nice! glad you added curry here, but let's import it like this

`import curry from 'ramda/src/curry';`

Reason being is that tree shaking doesn't work so easily/great yet. So if you import like you did here, the entire ramda library will be loaded instead of the just the one function we need.